src/types/Submissions.d.ts

```diff
@@ -14,7 +14,34 @@ type Submission = {
   metadataValidationStatus: ValidationStatus;
   fileValidationStatus: ValidationStatus;
   crossSubmissionStatus: CrossSubmissionStatus;
-  fileErrors: QCResult[]; // holds submission level file errors, e.g., extra files in S3 folder
+  /**
+   * The date and time when the validation process started.
+   *
+   * @note ISO 8601 date time format with UTC or offset e.g., 2023-05-01T09:23:30Z
+   */
+  validationStarted: string;
+  /**
+   * The date and time when the validation process ended.
+   *
+   * @note ISO 8601 date time format with UTC or offset e.g., 2023-05-01T09:23:30Z
+   */
+  validationEnded: string;
+  /**
+   * The last performed validation action scope.
+   *
+   * @see {@link ValidationTarget} for more information.
+   */
+  validationScope: ValidationTarget;
+  /**
+   * The last performed validation action type.
+   *
+   * @see {@link ValidationType} for more information.
+   */
+  validationType: Exclude<ValidationType, "All">[];
```

Comment:
Looks like type "All" isn't part of the `ValidationType` union anymore, I don't think the exclude is needed here

Reply:
Removed in 67834ff. Thanks for catching this.